plugins/ufm_consumer_plugin/conf/ufm_consumer_plugin.conf

```diff
@@ -0,0 +1,11 @@
+[common]
+port_number=8989
+ssl_apache_port=7443
+apache_port=8888
+
+
+[logs-config]
+logs_file_name = /ufm_consumer_plugin.log
```

Comment:
please verify full path location of the plugin log...

Reply:
removed